x-pack/solutions/security/plugins/security_solution/public/cloud_security_posture/components/csp_details/misconfiguration_findings_details_table.tsx

```diff
@@ -68,7 +68,7 @@ const getFindingsStats = (
         }
       ),
       count: failedFindingsStats,
-      color: euiThemeVars.euiColorVis9,
```

Comment:
`euiColorVis9 ` became yellow in Borealis. vis colors shouldn't be used for semantic purposes exactly for this reason - the vis color can change to pretty much anything. Changing to danger, while it is a different color, brighter one, and it will also change for Amsterdam. need confirmation from designers